bdk-ffi/src/psbt.rs

```diff
@@ -76,6 +79,48 @@ impl PartiallySignedTransaction {
     }
 }
 
+/// A key-value map for an input of the corresponding index in the unsigned
+/// transaction.
+#[derive(Debug)]
+pub(crate) struct Input {
+    _inner: BdkInput,
+}
+
+impl From<BdkInput> for Input {
+    fn from(input: BdkInput) -> Self {
+        Input { _inner: input }
+    }
+}
+
+/// A Signature hash type for the corresponding input. As of taproot upgrade, the signature hash
+/// type can be either [`EcdsaSighashType`] or [`SchnorrSighashType`] but it is not possible to know
+/// directly which signature hash type the user is dealing with. Therefore, the user is responsible
+/// for converting to/from [`PsbtSighashType`] from/to the desired signature hash type they need.
+#[derive(Debug)]
+pub(crate) struct PsbtSighashType {
+    inner: BdkPsbtSighashType,
```

Comment:
I like the idea of using `inner` as the variable name, and wonder if we should eventually refactor other structs that use the same design pattern (simple wrappers over a BDK types).